test/unit/vl-grid.test.html

```diff
@@ -137,6 +149,16 @@
         assert.isTrue(element.classList.contains('vl-grid--is-stacked'));
       });
 
+      test('heeft de vl-grid--is-stacked-large class', () => {
+        const element = fixture('vl-grid-is-stacked-large-fixture');
+        assert.isTrue(element.classList.contains('vl-grid--is-stacked-large'));
+      });
+
+      test('heeft de vl-grid--is-stacked-small class', () => {
+        const element = fixture('vl-grid-is-stacked-small-fixture');
+        assert.isTrue(element.classList.contains('vl-grid--is-stacked-small'));
+      });
```

Comment:
Zou eventueel ook met for loop over de stacked varianten kunnen, maar goed genoeg aangezien het er slechts 3 zijn.